app/components/Settings/Settings.react.js

```diff
@@ -62,7 +64,9 @@ class Settings extends Component {
 
     componentDidMount() {
         this.fetchData();
-        this.props.dispatch(Actions.hasCerts());
+        checkConnectorSettings = setInterval(() => {
+            this.props.dispatch(Actions.getConnectorSettings());
+        }, 2000);
```

Comment:
how could this change while the user is still on this page? won't they have saved their settings in the previous step before getting here?